crates/python/tests/test_api.py

```diff
@@ -1,11 +1,12 @@
 from unittest.mock import patch
 import pytest
 
+import re
+
 import qcs_sdk
 
 
 @pytest.mark.asyncio
-@pytest.mark.skip
```

Comment:
This seems unrelated to getting quilc version. Was this change intended?

Reply:
Yes, when I added a test for the new quilc method, I also updated `conftest.py` so that these tests will load the config from `tests/qcs_config`. That way the test would work without having a QCS config in `~/.qcs` and will work in CI. That change also allowed this test to run successfully on CI so I removed the `skip` decoration. 

The other tests with `skip` would require a valid QCS config so we can't run those on CI just yet.